docs/RUNNING-LOCALLY.md

```diff
@@ -12,13 +12,11 @@ Prerequisites
 - [NodeJS](https://nodejs.org) (at least v6)
 - [Python](https://www.python.org)
 - [jq](https://stedolan.github.io/jq/)
-- [Asar](https://github.com/electron/asar)
 - [Codespell](https://github.com/lucasdemarchi/codespell)
 - [curl](https://curl.haxx.se/)
 
 ### Windows
 
-- [Rimraf](https://github.com/isaacs/rimraf)
```

Comment:
Aha, I'd been wondering for ages if rimraf was actually unused, but never got round to actually testing it :)